saleor/order/utils.py

```diff
@@ -193,12 +193,13 @@ def update_taxes_for_order_line(
     line_price = line.unit_price.gross if tax_included else line.unit_price.net
     line.unit_price = TaxedMoney(line_price, line_price)
 
-    price = manager.calculate_order_line_unit(order, line, variant, product)
```

Comment:
What about second place where we call `manager.calculate_order_line_unit` ?
`add_variant_to_order` has logic
```
        unit_price = manager.calculate_order_line_unit(order, line, variant, product)
        line.unit_price = unit_price
        line.total_price = unit_price * quantity
        line.tax_rate = manager.get_order_line_tax_rate(
            order, product, variant, None, unit_price
        )
        line.total_price = unit_price * quantity
```